app/src/main/java/com/brainwallet/tools/util/LocaleHelper.kt

```diff
@@ -9,6 +9,7 @@ import androidx.preference.PreferenceManager
 import com.brainwallet.data.model.Language
 import java.util.Locale
 
+@Deprecated(message = "migrate using SettingRepository for persistence the user preference")
```

Comment:
Should we add and FirebaseAnalytics metrics test point to track when we can remove completely @andhikayuana ?
      
      
```
        pseudo  public static final String _20250225_DULUP = "did_use_legacy_user_preference";
        Bundle params = new Bundle();
        params.putDouble("sync_time_elapsed", minutesValue);
        params.putLong("sync_start_timestamp", startSync);
        params.putLong("sync_last_timestamp", lastSync);
        AnalyticsManager.logCustomEventWithParams(BRConstants._20250225_DULUP, params);
```